src/FileConventions.Test/DummyFiles/DummyWithoutMissingVersionsInNugetPackageReferences.fsx

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env -S dotnet fsi
+
+open System.IO
+open System
+
+#r "nuget: Fsdk, 0.6.0--date20230214-0422.git-1ea6f62"
```

Comment:
@realmarv this is wrong, this not the way to specify versions in nuget refs

Reply:
Done